install.markdown

```diff
@@ -3,37 +3,29 @@ layout: page
 title: Install IntelliJ Rust
 ---
 
-## Getting alpha builds
+## Alpha builds
 
-IntelliJ Rust is currently in the _alpha_ stage of development. When we reach some acceptable level of stability,
-we publish alpha builds to the IntelliJ's main plugin [repository](https://plugins.jetbrains.com/plugin/8182).
-You can download and install these builds directly from `Plugins > Browse repositories` screen in your IDE.
+IntelliJ Rust is in _alpha_ stage. We regularly publish stable builds to the JetBrains plugin [repository](https://plugins.jetbrains.com/plugin/8182).
+You can download and install these builds directly from `Settings / Preferences | Plugins | Marketplace` in your IDE.
 
-As this is a preview release, something might go wrong. So, be brave to face some nasty bugs.
-If you find one, we would very appreciate if you file an [issue](https://github.com/intellij-rust/intellij-rust/issues)
-on our bugtracker or ping us on our [chat](https://gitter.im/intellij-rust/intellij-rust).
+For bug reports and feature requests, please use our [issue tracker](https://github.com/intellij-rust/intellij-rust/issues).
 
-## Setting up nightly builds
+## Nightly builds
 
-In addition to preview releases, we also
-ship pre-release builds of _intellij-rust_ and _intellij-toml_ in the nightly channel.
+In addition to aplha releases, we also ship pre-builds of _intellij-rust_ and _intellij-toml_ in the nightly channel.
```

Comment:
```suggestion
In addition to stable releases, we also ship pre-builds of _intellij-rust_ in the nightly channel.
```

Not sure about `stable` here but it's more suitable than `alpha`. Maybe one of `general`, `common`, `main` is better.
Also, we don't publish nightly builds for _intellij-toml_ plugin anymore (they don't make a lot of sense) so I think we can drop its mention